admin/sql/updates/2019-07-09-add-recommendation-table.sql

```diff
@@ -0,0 +1,65 @@
+BEGIN;
+
+CREATE SCHEMA recommendation;
+
+CREATE TABLE recommendation.cf_recording (
+  id                  SERIAL, -- PK
+  user_id             INTEGER NOT NULL, --FK to "user".id
+  recording_msid      UUID NOT NULL,
+  type                cf_recording_type,
```

Comment:
cf_recording_type isn't created in this script.

Reply:
<3